packages/react/src/components/UIShell/SideNav.js

```diff
@@ -90,17 +91,24 @@ const SideNav = React.forwardRef(function SideNav(props, ref) {
     });
   }
 
+  const eventHanders = {
+    onFocus: event => handleToggle(event, true),
+    onBlur: event => handleToggle(event, false),
+  };
+
+  if (addMouseListeners) {
+    eventHanders.onMouseEnter = () => handleToggle(true, true);
```

Comment:
it looks like this is carried over from previous PRs but I just want to confirm, `handleToggle` accepts an event object or a boolean as its first arg?

Reply:
Hi @emyarod - yes that is right. That was also the case before my changes in this area went in yesterday: https://github.com/carbon-design-system/carbon/commit/7436b9842d809469a2a200a2f29c37755e7e7b50#diff-bc8465da0af4192df7e3a6fc671cb5c0 . In https://github.com/carbon-design-system/carbon/commit/92076fdaf57bfaa2b5745d750ac73c59709c1687#diff-bc8465da0af4192df7e3a6fc671cb5c0 I also added the second boolean, so the `expandedViaHoverState` can also be updated under the right conditions.